scripts/create-test-cassandra-db.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Spins up a test cassandra docker container and loads it with data.
+#
+# By default, the container will start up a 'default' database an not require a
+# username or password.
+#
+# Requires `cassandra`: <https://cassandra.apache.org/_/quickstart.html>
```

Comment:
Do we have this on the action runner?

Reply:
it just requires docker. will update the comment.